test/integration/scheduler/volume_binding_test.go

```diff
@@ -571,6 +583,22 @@ func validatePVCPhase(t *testing.T, client clientset.Interface, pvcName string,
 	}
 }
 
+func validatePVCEvent(t *testing.T, client clientset.Interface, pvcName string, ns string, reason string) {
+	claim, err := client.CoreV1().PersistentVolumeClaims(ns).Get(pvcName, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Failed to get PVC %v/%v: %v", ns, pvcName, err)
+	}
+
+	pvcEvents, err := client.CoreV1().Events(ns).Search(legacyscheme.Scheme, claim)
+	if err != nil {
+		t.Errorf("Failed to get PVC events %v/%v: %v", ns, pvcName, err)
+	}
+
+	if len(pvcEvents.Items) > 0 && pvcEvents.Items[0].Reason != reason {
```

Comment:
I think it's possible for PVCs to have multiple events.  So I think we should iterate through all the events and find one that matches.

Reply:
Done